plugins/services/src/js/containers/pod-debug/PodDebugContainer.js

```diff
@@ -41,8 +40,7 @@ class PodDebugTabView extends React.Component {
     const {pod} = this.props;
     const queue = pod.getQueue();
 
-    if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(pod)
```

Comment:
If we are not using `shouldDisplayDeclinedOffersWarning` anywhere I think we should remove it